outlook/mail/tool.gpt

```diff
@@ -103,26 +103,18 @@ Param: destination_folder_id: The ID of the folder to move the message into.
 
 #!${GPTSCRIPT_TOOL_DIR}/bin/gptscript-go-tool moveMessage
 
----
-Name: Get Default Timezone
-Description: Get the default timezone for the user.
-Credential: ./credential
-
-#!${GPTSCRIPT_TOOL_DIR}/bin/gptscript-go-tool getDefaultTimezone
-
 ---
 Name: Outlook Mail Context
 Type: context
-Share Context: Current Date and Time from ../../time
-Share Tools: Get Default Timezone
+Share Context: ../../time
 
 #!sys.echo
 
-## Instructions for using the Microsoft Outlook Mail tools
+# START INSTRUCTIONS: Microsoft Outlook Mail tools
 
 You have access to tools for the Microsoft Outlook Mail API.
 
-Before calling any other Outlook tools, call Get Default Timezone tool, so that you know the user's timezone.
+Display all dates and times in the user in the user's preferred timezone. When the user gives values for dates and times, assume they are in the user's preferred timezone unless otherwise specified by the user.
```

Comment:
```suggestion
Display all dates and times in the user's preferred timezone. When the user gives values for dates and times, assume they are in the user's preferred timezone unless otherwise specified by the user.
```

Reply:
fixed